### DISCUSSION TITLE
discord invite invalid

### DISCUSSION THREAD

**Discussion by 3a1b2c3** [reworkd/AgentGPT#463] [Q&A] [answered]
Great project, are you on slack or discord?

**Comment by 3a1b2c3** [comment]
![image](https://user-images.githubusercontent.com/74843139/236657082-56412c1e-0cd6-4d31-8949-b8c8e78ff7af.png)

**Comment by awtkns** [comment, marked as answer]
Hi @3a1b2c3 you can join the discord community [🔗 here](https://agentgpt.reworkd.ai/)! Both the readme and the website have working links. Where did you find the broken link?
